main/inc/lib/main_api.lib.php

```diff
@@ -4662,7 +4664,8 @@ function api_get_status_langvars() {
         SESSIONADMIN    => get_lang('SessionsAdmin', ''),
         DRH             => get_lang('Drh', ''),
         STUDENT         => get_lang('Student', ''),
-        ANONYMOUS       => get_lang('Anonymous', '')
+        ANONYMOUS       => get_lang('Anonymous', ''),
+        STUDENT_BOSS       => get_lang('RoleStudentBoss', '')
```

Comment:
It's weird in an academical context, but not in a corporate context. It could also be called EMPLOYEE_SUPERIOR but it's just a name. But yeah... STUDENT_BOSS is a funny name (but it is short)